src/index.js

```diff
@@ -7,32 +7,33 @@ import PageLoading from './Components/PresentationalComponents/Snippets/PageLoad
 import { EnvironmentContext } from './App';
 import { useHccEnvironmentContext } from './Helpers/Hooks';
 import { AccountStatsContextWrapper } from './Utilities/AccountStatsContextWrapper';
+import ReducerRegistry from './Utilities/ReducerRegistry';
 
 const WrappedSystemCves = ({ getRegistry, ...props }) => {
-    const [Wrapper, setWrapper] = useState();
+    const [store, setStore] = useState(ReducerRegistry.getStore());
```

Comment:
Instead of a `useState` we can just use a `useRef` like here in compliance: https://github.com/RedHatInsights/compliance-frontend/blob/master/src/Modules/ComplianceDetails.js#L11